server/views/pages/managementReporting/index.njk

```diff
@@ -0,0 +1,44 @@
+{% extends "partials/layout.njk" %}
+
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% set title = "Management reporting" %}
+
+{% block pageTitle %}{{ 'Management reporting - ' + applicationName }}{% endblock %}
+
+{% block heading %}
+  <h1 class="govuk-heading-l">{{ title }}</h1>
+{% endblock %}
+
+
+{% block content %}
+
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row govuk-!-margin-bottom-0">
+      <div class="govuk-grid-column-two-thirds">
+
+
+        {% if definitions.length > 0 %}
+        {% for definition in definitions %}
+          {% if definition.authorised %}
+            <h2 class="govuk-heading-m">{{ definition.name }}</h2>
+            <p class="govuk-body">{{ definition.description }}</p>
+
+            {% for variant in definition.variants %}
+              <h3 class="govuk-heading-s"><a id="{{ definition.id }}-{{ variant.id }}"
+                                             href="/management-reporting/{{ definition.id }}-{{ variant.id }}"
+                                             class="govuk-link"> {{ variant.name }} </a></h3>
+              <p class="govuk-body">{{ variant.description }}</p>
+
+            {% endfor %}
+          {% endif %}
+        {% endfor %}
+        {% else %}
+          <p class="govuk-body">No management reports available</p>
```

Comment:
Reports are role controlled by the application and so only users with the right privs will get offered the report